public/js/clients/firefox.js

```diff
@@ -146,6 +146,45 @@ function createFrame(frame) {
 }
 
 const CALL_STACK_PAGE_SIZE = 25;
+const NEW_SOURCE_IGNORED_URLS = ["debugger eval code", "XStringBundle"];
+
+function makeDispatcher(client, actions) {
+  return {
+    paused(_, packet) {
+      // If paused by an explicit interrupt, which are generated by the
+      // slow script dialog and internal events such as setting
+      // breakpoints, ignore the event.
+      if (packet.why.type === "interrupted" && !packet.why.onNext) {
+        return;
+      }
+
+      // Eagerly fetch the frames
+      client.getFrames(0, CALL_STACK_PAGE_SIZE, res => {
+        actions.loadedFrames(res.frames.map(createFrame));
+      });
```

Comment:
Ah. You are reviewing on your phone?! Don't feel like you need to, go enjoy the conference :)